util/check-format-test.c

```diff
@@ -20,9 +20,9 @@
 
 /* For each of the following set of lines the tool should complain */
 /*@ tab character: 	 */
-/*@ cr character: 
 */
+/*@ carriage return character: 
 */
```

Comment:
I wonder what happens with this on macos, which classically has CR as line termination character...

Reply:
Not that it matters much... Perhaps it's enough with a warning somewhere that this won't be properly detected on all platforms

Reply:
CR characters are orthogonal to line termination of macros. Multi-line macro definitions use a backslash `\` at/near line end, and these are already properly handled by the tool.

On Windows systems, line ends have a CR just before the LF character, while on Unix-like systems only LF is used. The tool so far complained about all CR characters, which was not suitable for line ends on Windows. I've changed this such that only intra-line CRs are reported, which - just like TAB and FF characters - should be avoided in source code (though from the C perspective these are just other forms of whitespace) regardless of the platform.